src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -47,21 +40,89 @@ void createStation() {
                         .when().get("/stations")
                         .then().log().all()
                         .extract().jsonPath().getList("name", String.class);
-        assertThat(stationNames).containsAnyOf("강남역");
+        assertThat(stationNames).containsAnyOf(STATION_NAME_GANGNAM);
     }
 
     /**
      * Given 2개의 지하철역을 생성하고
      * When 지하철역 목록을 조회하면
      * Then 2개의 지하철역을 응답 받는다
      */
-    // TODO: 지하철역 목록 조회 인수 테스트 메서드 생성
+    @DisplayName("지하철 역 목록을 조회한다.")
+    @Test
+    void showStationsTest() {
+        //given
+        createStation(STATION_NAME_GANGNAM);
+        createStation(STATION_NAME_YEOGSAM);
+
+        //when
+        ExtractableResponse<Response> response = showStations();
+
+        //then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+
+        //then
+        List<String> stations = response.jsonPath().getList("name", String.class);
+        assertThat(stations).contains(STATION_NAME_GANGNAM);
+        assertThat(stations).contains(STATION_NAME_YEOGSAM);
+        assertThat(stations).doesNotContain("존재하지않는역");
+    }
+
 
     /**
      * Given 지하철역을 생성하고
      * When 그 지하철역을 삭제하면
      * Then 그 지하철역 목록 조회 시 생성한 역을 찾을 수 없다
      */
-    // TODO: 지하철역 제거 인수 테스트 메서드 생성
+    @DisplayName("지하철 역을 삭제한다")
+    @Test
+    void removesStationTest() {
+        //given
+        Long gangnamId = createStation(STATION_NAME_GANGNAM).jsonPath().getLong("id");
+        Long yeogsamId = createStation(STATION_NAME_YEOGSAM).jsonPath().getLong("id");
```

Comment:
이거는 호불호가 나뉘는 내용일 수 있지만 언급해보자면
변수 네이밍을 한글로 해보는 것도 색다른 시도일 수 있습니다!
참고만 해주세요~

```suggestion
        Long 강남역 = createStation(STATION_NAME_GANGNAM).jsonPath().getLong("id");
        Long 역삼역 = createStation(STATION_NAME_YEOGSAM).jsonPath().getLong("id");
```